index.html

```diff
@@ -230,6 +230,46 @@ <h2 class="headline inter-font">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
     </section>
+    <section class="form-section inter-font">
+      <form action="https://formspree.io/f/xqkjapyw" method="post" class="formSection">
+      <input type="text" placeholder="Full name" maxlength="30" id="name" name="name"
+      required>
+      <input type="email" name="email" placeholder="Email Address"  id="email" required>
+      <textarea  placeholder="Enter your text here"  maxlength="500" id="msg"  name="msg" required></textarea>
+      <button class="seeProject inter-font" type="submit" name="touch">Get in touch</button>
+  
+    </form>
+    <ul class="socialMedia">
+      <li class="twitter">
+        <a href="#">
+          <img src="/assets/images/twitter.svg" alt="twitter image" />
+        </a>
+      </li>
+      <li class="linkedin">
+          <a href="#"> 
+              <img src="/assets/images/linkedin.svg" alt="linkedin image" />
+          </a>
+      </li>
+      <li class="meduim">
+          <a href="#">
+        <img src="/assets/images/medium.svg" alt="medium image" />
+      </a>
+      </li>
+      <li class="github">
+          <a href="#">
+        <img src="/assets/images/github.svg" alt="github image" />
+          </a>
+      </li>
+      <li class="angleList">
+          <a href="#">
+        <img src="/assets/images/angellist.svg" alt="angleList image" />
+        </a>
+      </li>
+    </ul>
+    </section>
+    <footer>
+      <a href="#">myemailhere@email.com</a>
+    </footer>
   </body>
```

Comment:
[OPTIONAL]: Please add your actual email address for this is your own portfolio web page you are building.